src/main/java/com/spring/app/dto/CountryProfileDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.spring.app.dto;
+
+import com.spring.app.EconomicLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+
+/**
+ * Data transfer object for storage info about country's profile
+ *
+ * @author Lyubov Ruzanova
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CountryProfileDTO implements Serializable {
+    private Long numberOfVacancies;
```

Comment:
Some sacral sense is enclosed using number Wrapper instead of simple number?

Reply:
now it does not matter) understood, I will use a primitive type